spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/resource/ResourceController.java

```diff
@@ -67,11 +68,19 @@ public ResourceController(ResourceRepository resourceRepository,
 		this.helper.setAlwaysUseFullPath(true);
 	}
 
-	@RequestMapping("/{name}/{profile}/{label}/**")
 	public String resolve(@PathVariable String name, @PathVariable String profile,
```

Comment:
I was just trying to retain backward compatibility for anything explicitly using the classes and methods (i.e. trying to prevent breaking existing unit tests).  If you want to get rid of this method, that's fine. it probably also makes sense to get rid of the other `resolve` method as well.